File: app/src/main/java/cz/kutner/comicsdb/connector/service/ForumService.java
package cz.kutner.comicsdb.connector.service;

import java.util.List;

import cz.kutner.comicsdb.model.ForumEntry;
import retrofit.http.GET;
import retrofit.http.Query;
import rx.Observable;

public interface ForumService {
    @GET("/forum.php")
    Observable<List<ForumEntry>> filteredForumList(@Query("str") int page, @Query("id") int forumId, @Query("val") String searchText);
}

File: app/src/main/kotlin/cz/kutner/comicsdb/connector/service/ForumService.kt
package cz.kutner.comicsdb.connector.service

import cz.kutner.comicsdb.model.ForumEntry
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ForumService {
    @GET("/api.php?get=forum")
    fun filteredForumList(@Query("start") start: Int, @Query("records") records: Int): Call<List<ForumEntry>>
}
